### ISSUE TITLE
Documenting Prismic components to find out if we can remove some

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#9917]
## Description
By working on Visual Stories and discussing the possibility of creating new components (or modifying existing ones), we tended to fall into rabbit holes when talking about existing one. It was also discussed that their purpose wasn't always clear and that documentation would be useful moving forward.

I created [a document to analyse](https://docs.google.com/presentation/d/1RytMltnIDQ44BXEa8JR_2-hXslaLEogo49jopO3qLuY/edit?usp=sharing) what we have, how it is used, and started suggesting what could be removed or tweaked to be more useful. 

This will be a work in progress for a bit, but the goal is to have tickets come out of it to streamline our Prismic component offer.

## Definition of Done:
- Cleaned up slices as much as possible
- Created a permanent document describing each slice and what they are to be used for/what they look like.

**Comment by rcantin-w** [comment]
Have booked a meeting with @jennpb on Monday to discuss what is left to answer in the document.

**Comment by rcantin-w** [comment]
Looking for a first round of review on [my Gitbook documentation page](https://app.gitbook.com/o/-LumfFcEMKx4gYXKAZTQ/s/451yLOIRTl5YiAJ88yIL/~/changes/3/custom-slices). Not putting the ticket down as ready for review as I still have some cleaning up to do (see first point in DoD) and extra editing on the documentation.

**Comment by rcantin-w** [comment]
The last of the last:
- Waiting on #9868 
- Need a better description for Title Text List ( @jennpb has it on her to do, but if anyone else has a good one)
- Do we want to rename Title Text List on the UI or are we happy with that? It's used only once, here https://wellcomecollection.org/pages/YDaP2BMAACUAT7DS

**Comment by alexwlchan** [comment]
> Do we want to rename Title Text List on the UI or are we happy with that?

Maybe "List of links"? That describes what we're actually using it for.

(And maybe trim some of the options while we're here? If you look at the current [Prismic model definition](https://wellcomecollection.org/pages/YDaP2BMAACUAT7DS), there are several options we aren't using in the one page we use it on. There are no labels, no heading4/list-item, and nothing but single lines of text.

**Comment by rcantin-w** [comment]
Thanks @alexwlchan !

1. It's like "List of links with description" though, not just a bullet point of links 🤔 I don't have a clever idea though
2. Agree tags isn't used, maybe it was for ["online" or "library"](https://github.com/wellcomecollection/wellcomecollection.org/issues/6010#issuecomment-775104643)? @jennpb do you remember and do you think we could remove?
3. What do you mean by "heading4/list-item"?

**Comment by alexwlchan** [comment]
heading4/list-item is what I meant to be linking to, from the Prismic model definition:

https://github.com/wellcomecollection/wellcomecollection.org/blob/a94bf4487e3d7701796bcc68ae265209c3615b20/prismic-model/src/parts/body.ts#L141-L150

I don't think we need those `extraTextOptions`.

**Comment by rcantin-w** [comment]
Ahh yes I see! @jennpb any objections in regards to how it was first designed?

**Comment by jennpb** [comment]
> Agree tags isn't used, maybe it was for https://github.com/wellcomecollection/wellcomecollection.org/issues/6010#issuecomment-775104643? @jennpb do you remember and do you think we could remove?

It was for that purpose, yes. It looks like the content editor has decided to separate the links out into separate sections instead of using the labels, so I think it's fine to remove labels.

**Comment by rcantin-w** [comment]
what about `Descriptive links list`?

**Comment by jennpb** [comment]
As far as I can tell, the prismic model still meets the original intent as described here: https://github.com/wellcomecollection/wellcomecollection.org/issues/6010 - there are examples that we'd worked up with the content team at that point.

`Descriptive links list` is fine. I still think that bulleted text should be allowed in it.

**Comment by rcantin-w** [comment]
Do you mean something like this?

![Image](https://github.com/wellcomecollection/wellcomecollection.org/assets/110461050/fc8a475e-e0e8-4181-8244-ab0b5d867fd5)

**Comment by jennpb** [comment]
More like this (sorry just doing this in a doc)
<img width="393" alt="Screenshot 2023-06-29 at 11 30 45" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/1446139/6439ee93-cb59-4846-bd5b-8f1fe1c3ccd9">

**Comment by rcantin-w** [comment]
Moving to Blocked until https://github.com/wellcomecollection/wellcomecollection.org/issues/9868 is unblocked.

Clearing out Image List is my last item in this, the only outlier being Media object list which depends on more work on new slices. It has its own ticket in the backlog as #9849 .
